GVRf/Framework/framework/src/main/java/org/gearvrf/GVRScene.java

```diff
@@ -445,13 +448,22 @@ public void bindShaders(GVRSceneObject root) {
      * @see GVRScene.getLightList
      */
     protected boolean addLight(GVRLightBase light) {
```

Comment:
Btw, since people could derive from GVRScene addLight should be either final or package private. Can't see anything good coming out of end users messing with this method.